Source/Documentation/Manual/news.rst

```diff
@@ -32,6 +34,7 @@ What's been added
 What's been improved
 --------------------
 
+- superelevation
```

Comment:
Superelevation is already mentioned, see line 46

Reply:
Yes, of course.
Thanks for spotting that.